docs/src/App.vue

```diff
@@ -1,11 +1,55 @@
 <template>
   <div id="q-app">
     <router-view />
+
```

Comment:
Why all this code when you could use a `this.$q.notify()` call instead?
Cookies are also NOT needed. This can be handled with LocalStorage to avoid increasing the length of each HTTP request (which will include this cookie).

Also, in this example, the QBanner goes under the sidebars and footer. `z-max` would have been more appropriate.